docs/source/reference/glossary.rst

```diff
@@ -1,7 +1,73 @@
 Glossary
 ========
 
+
+API
+---
+
+application programming interface
+
+CLI
+---
+
+command line interface
+
+CSF/CSF3
```

Comment:
Not sure if CSF is mentioned in the public docs or only in the internal ones, so this might not be relevant here.

Reply:
I'd prefer to keep CSF out of the public-oriented MatFlow docs. Is it mentioned somewhere else?